htdocs/core/class/html.form.class.php

```diff
@@ -4903,8 +4903,8 @@ public function formconfirm($page, $title, $question, $action, $formquestion = '
                          			if ($("input[name=\'" + inputname + "\']").attr("type") == "radio") {
 										inputvalue = $("input[name=\'" + inputname + "\']:checked").val();
 									} else {
-                         		    	if ($("#" + inputname).attr("type") == "checkbox") { more = ":checked"; }
-                         				inputvalue = $("#" + inputname + more).val();
+                         		    	if ($("input[name=\'" + inputname + "\']").attr("type") == "checkbox") { more = ":checked"; }
```

Comment:
@hregis Be careful ! select boxes may also be used in formconfirm and this will not work !
You should write $("[name=xxx]") without the input tag.

Reply:
@altairis-tof 
and if your remove "input" in `"input[name=\'" + inputname + "\']" `?

```
if ($("[name=\'" + inputname + "\']").attr("type") == "checkbox") { more = ":checked"; }
inputvalue = $("[name=\'" + inputname + "\']" + more).val();
```

Reply:
yes it works; dont forget to do the same on each line you changed ;)

Reply:
i see you did it ! perfect !